README.md

```diff
@@ -1,5 +1,14 @@
 # Solana Certora Verification Tool (CVT) #
 
-This library defines a set of functions and macros that can be called from Solana
+This workspace defines a set of functions and macros that can be called from Solana
 programs to create non-deterministic environments, assume/assert, and
 alternative (simpler but complete) implementations for std containers such as `Vec` class.
+
+It includes the following crates:
+- `anchor` [TODO: description]
+- `containers` [TODO: description]
+- `cvt_core` [TODO: description]
```

Comment:
I think this crate should still be `cvt`. Having to type `cvt_core` everywhere is too much